modules/account-lib/test/unit/coin/dot/transaction.ts

```diff
@@ -51,4 +60,125 @@ describe('Dot Transaction', () => {
       should.deepEqual(tx.transactionSize(), DotResources.rawTx.transfer.unsigned.length / 2);
     });
   });
+
+  describe('Dot Explain Transaction', () => {
+    let factory: StubTransactionBuilderFactory;
+    const { rawTx } = DotResources;
+    const sender = DotResources.accounts.account1;
+    const receiver = DotResources.accounts.account3;
+    const proxySender = DotResources.accounts.account3;
+
+    beforeEach(() => {
+      factory = register('dot', StubTransactionBuilderFactory);
+    });
+
+    it('should explain a transfer transaction', async () => {
```

Comment:
This test  throwing an unhandled exception and failing if i run it alone. But pass if i run the whole Dot Explain Transaction.

Reply:
Strange, working on my system

Reply:
Ignore my comment, is probably webstorm not loading the config correctly.